File: app/src/main/java/com/example/pokedex/model/PrevEvolution.java
package com.example.pokedex.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PrevEvolution {
    @SerializedName("num")
    @Expose
    private String num;
    @SerializedName("name")
    @Expose
    private String name;
}

File: app/src/main/java/com/example/pokedex/model/PrevEvolution.kt
package com.example.pokedex.model

import com.google.gson.annotations.Expose

import com.google.gson.annotations.SerializedName


class PrevEvolution {
    @SerializedName("num")
    @Expose
    val evoLevel: String? = null

    @SerializedName("name")
    @Expose
    val name: String? = null
}